my-app/src/component/TodoButton/TodoButton.jsx

```diff
@@ -0,0 +1,12 @@
+import React from "react";
+
+//関数「Button」を定義しボタン操作の動きを設定
```

Comment:
```suggestion
/**
 * ボタンコンポーネント
 * @param onClick クリックで発火する関数
 * @param label ボタン内の文字
 * @param className クラス名
 * @param type ボタンのタイプ
 */
```

コンポーネントに関するコメントは上記のようにJS Docを使用しましょう。
何のコンポーネントなのか、propsは何かを記載して下さい。
上記のように記載することでvsコード上で動画のようにpropsが表示されるようになります。

https://github.com/shiiyuho/react-study/assets/85776582/eb3ea451-2036-4c57-9073-3236ae4a09c7